### PR TITLE
Use a driver supporting JavaScript in system tests

### PR DESCRIPTION
**Pull request by javierm**
## References

* System tests were introduced in pull request #3621

## Objectives

* Test the way the application behaves in browsers supporting JavaScript (about 98% of existing browsers)
* Use selenium/chromedrive as the default driver, so we don't have to remember to add a `:js` tag on every test we write
* Make usability issues more obvious in system tests

## Notes

In the past, using a headless browser in tests made the test suite much slower because we had to use database cleaner to truncate the database after each test. With system tests, however, we use database transactions to make sure data is removed between tests, and so the speed difference isn't that big.

### CODE REVIEW

File: spec/shared/system/nested_imageable.rb [hunk 1]
```diff
@@ -57,7 +57,7 @@
       expect_image_has_title("clippy.jpg")
     end
 
-    scenario "Should not update nested image file title with file name after choosing a file when title already defined", :js do
+    scenario "Should not update nested image file title with file name after choosing a file when title already defined" do
```
[the remark this answers is not in the document]
Comment (javierm) [reply spec/shared/system/nested_imageable.rb:60]: It's shorter than it was :wink:.

File: spec/system/admin/poll/shifts_spec.rb [hunk 2]
```diff
@@ -88,7 +88,7 @@
     end
   end
 
-  scenario "Vote Collection Shift and Recount & Scrutiny Shift don't include already assigned dates to officer", :js do
+  scenario "Vote Collection Shift and Recount & Scrutiny Shift don't include already assigned dates to officer" do
```
[the remark this answers is not in the document]
Comment (javierm) [reply spec/system/admin/poll/shifts_spec.rb:91]: It's shorter than it was :wink:.

File: spec/shared/system/nested_imageable.rb [hunk 3]
```diff
@@ -45,7 +45,7 @@
       expect(page).to have_selector ".file-name", text: "clippy.jpg"
     end
 
-    scenario "Should update nested image file title with file name after choosing a file when no title defined", :js do
+    scenario "Should update nested image file title with file name after choosing a file when no title defined" do
```
[the remark this answers is not in the document]
Comment (javierm) [reply spec/shared/system/nested_imageable.rb:48]: It's shorter than it was :wink:.

**Review by Senen — APPROVED** [review]
👏🏼

**Pull request merged**